Library/Feature.swift

```diff
@@ -1,5 +1,7 @@
 import Foundation
 
+// TODO: remove nativeCheckout, nativeCheckoutPledgeView once we've refactored FeatureFlagToolsViewModelTests
```

Comment:
The tests in `FeatureFlagToolsViewModelTests` depend on the native checkout feature flags being defined. We would need to refactor that VM to either run its tests on different flags (although this will change every time we add/remove feature flags) or abstract it in some way to not be dependent on the concrete flags. The tests are mostly just proving that we order the flags alphabetically.